src/Autocomplete/assets/src/controller.ts

```diff
@@ -50,7 +50,7 @@ export default class extends Controller {
 
         // multiple values excepted if this is NOT A select (i.e. input) or a multiple select
         const isMultiple = !this.selectElement || this.selectElement.multiple;
-        if (!this.formElement.disabled && !isMultiple) {
+        if (!this.formElement.required && !this.formElement.disabled && !isMultiple) {
```

Comment:
@weaverryan how do i update the dist js?

Reply:
`yarn build?` it seems this hasn't been for for a few PR's touching this file.
who should be doing this?

Reply:
Yup, `yarn build`. Sometimes we forget to run it for a PR - we have a CI job that should fail if you forget, but at the moment, it's failing for a different reason #427

Reply:
i'm on vacation for 2 weeks now. if you want to pick this up in my absense, feel free.

Reply:
I was too - I'll wait for you to get back - I think this PR just needs minor tweaks